frameworks/hdfs/tests/test_sanity.py

```diff
@@ -148,10 +147,23 @@ def test_kill_data_node():
 @pytest.mark.sanity
 @pytest.mark.recovery
 def test_kill_scheduler():
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+    scheduler_name = sdk_utils.get_scheduler_name(foldered_name)
+    scheduler_id = sdk_tasks.get_task_ids("marathon", scheduler_name)
```

Comment:
Strictly speaking, `scheduler_id` is a list of tasks, and we should use `scheduler_ids` as a variable name. I was initially confused because I read . the call to `check_tasks_updated` as passing in a single string, which would have been an error.